esi_leap/resource_objects/dummy_node.py

```diff
@@ -35,6 +35,11 @@ def get_project_id(self):
             node_dict = json.load(node_file)
         return node_dict.get("project_id", None)
 
+    def get_node_config(self):
+    	with open(self._path) as node_file:
+            node_dict = json.load(node_file)
+        return node_dict.get("server_config", None)
+
```

Comment:
Let's add this method to test_node.py and ironic_node.py as well. For the former it can return an empty dict; for the latter, it can also return an empty dict along with a comment mentioning that it'l lbe updated later.